modules/ting_fulltext/js/ting_fulltext.js

```diff
@@ -0,0 +1,20 @@
+(function ($) {
+  "use strict";
+
+  Drupal.behaviors.fullTextReview = {
+    attach: function (context) {
+      var selector = '.js-ting-lektor-fulltext';
+      $(selector, context).once('js-ting-lektor-fulltext', function () {
+         var url = $(this).attr("data-fulltext-url");
+         $.ajax({
+           type: 'POST',
+           url: url,
+           success: function (result) {
+             $(".js-ting-lektor-fulltext").html(result);
+           }
```

Comment:
This approach to using behaviors does not get the concepts of ids, context and selectors quite right.

1. Because we cannot know if there are multiple fulltext lektørudtalelser on a page we cannot use an HTML id to identify or select them. HTML IDs must be unique. Instead this PR currently uses a `js-` prefixed classname. This is good.
2. Drupal behaviors work within a context. This may be the entire page on the initial pageload or elements that are added at a later point in time through AJAX.  This PR currently takes the context into account when looking for new  elements to process. This is good.
3. A behavior may operate multiple times on the same element. This may not be a good thing(tm) and can be avoided using `once()` as is the case in this PR. This is good.
4. After completing the AJAX request to retrieve a lektørudtalelse the content should be inserted into the element for which it has been retrieved. This PR inserts the content into any HTML element matching the selector. This is a problem if there are multiple elements on the same page. The content should only be inserted into the element of which the content was retrieved. This PR inserts the content into all elements matching the selector. This is bad.